### PR TITLE
[SPH-TASK]Markup-Redone-Pages

### PR DESCRIPTION
**Pull request by ENRexGeorge**
Routes:
Home - localhost:3000/
Register Page - /register
Register Success Page - /register-success
Log In Page - /login

image
image
image
image

### CODE REVIEW

**Comment by biancanailon** [comment]
This works fine to me but the design is quite different on my side.

Update: The design is now same on my side

image
image
image
image

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,36 @@
 /be/compendium/db/migrations/__pycache__
 /be/venv
 /be/.venv
+<<<<<<< Updated upstream
```
Comment (roseaugusto) [inline .gitignore]: naay wa pa na fix na conflict here @ENRexGeorge

Comment (ENRexGeorge) [reply .gitignore]: okay miss, on it

**Review by roseaugusto — APPROVED** [review]
Approved!

**Pull request merged**